.rubocop.yml

```diff
@@ -0,0 +1,29 @@
+Lint/UnusedMethodArgument:
+    AllowUnusedKeywordArguments: true
```

Comment:
Ah ok. Can you please disable it locally instead of project wide then?
http://www.rubydoc.info/gems/rubocop/0.7.1#Disabling_Cops_within_Source_Code